dts/bindings/clock/atmel,saml2x-gclk.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2021 Argentum Systems Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Atmel SAML2x Generic Clock Controller (GCLK)
+
+compatible: "atmel,saml2x-gclk"
+
+include: [clock-controller.yaml, base.yaml]
+
+properties:
+    reg:
+      required: true
+
+    "#clock-cells":
+      const: 1
+
+clock-cells:
+  - periph_ch
```

Comment:
Why isn't this `periph-ch`?

Reply:
I can't answer that one, but I'm following suit from various drivers and [`dts/bindings/clock/atmel,samd5x-gclk.yaml:18`](https://github.com/zephyrproject-rtos/zephyr/blob/88ca3aca98c0788e2c45db5c3fb9c8523293a8c0/dts/bindings/clock/atmel,samd5x-gclk.yaml#L18)

... `periph-ch` doesn't exist as a string in the code base.